cpp/src/io/orc/orc_gpu.hpp

```diff
@@ -102,21 +102,22 @@ struct DictionaryEntry {
 struct ColumnDesc {
   uint8_t const* streams[CI_NUM_STREAMS];  // ptr to data stream index
   uint32_t strm_id[CI_NUM_STREAMS];        // stream ids
-  uint32_t strm_len[CI_NUM_STREAMS];       // stream length
+  uint32_t strm_len[CI_NUM_STREAMS];       // stream length TODO check for 64
```

Comment:
Humnn, stripe sizes in reality are only in O(megabytes) thus I don't think it is necessary here to have 64 bit.

Reply:
`Stream::length` in specs is `uint64_t`, so this one should probably be as well
I think we can't assume upper limit on row count or size of a stripe, since there could a degenerate case with a very wide nesting, leading to a huge stripe.